### ISSUE TITLE
0.5.17 Breaks Yarn PnP installations

### ISSUE THREAD

**Issue by Mike-Dax** [evanw/node-source-map-support#272]
```
WARNING in ./.yarn/cache/source-map-support-npm-0.5.17-afcc0d09fa-2.zip/node_modules/source-map-support/source-map-support.js
Module not found: Error: A package is trying to access another package without the second one being listed as a dependency of the first one

Required package: worker_threads (via "worker_threads")
Required by: source-map-support@npm:0.5.17 (via /~/test/.yarn/cache/source-map-support-npm-0.5.17-afcc0d09fa-2.zip/node_modules/source-map-support/source-map-support.js)
```

I believe this is due to the dynamic require of worker_threads.

I'm not sure if it's possible to avoid a dynamic require in this package or if Yarn PnP should support dynamic requires of inbuilt Node modules.

**Comment by LinusU** [comment]
Hmm, would you mind testing if #271 fixes this for you? ☺️

**Comment by LinusU** [comment]
Fixed in 0.5.18
